circle.yml

```diff
@@ -5,7 +5,7 @@ deployment:
   production:
     branch: master
     heroku:
-      appname: hylo-node
+      appname: hylo-node-hitfin
```

Comment:
This actually wasn't suppose to be in the pull request. Good to know that there is an environment variable that we can use